source/fuzz/transformation_record_synonymous_constants.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2020 Stefano Milizia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "transformation_record_synonymous_constants.h"
+
+namespace spvtools {
+namespace fuzz {
+
+namespace {
+static inline bool IsStaticZeroConstant(
```

Comment:
Remove `static` - it's redundant since you're in an anonymous namespace.

Remove `inline` - the code isn't performance critical.

Change name to `IsScalarZeroConstant`.